README.md

```diff
@@ -1886,6 +1886,10 @@ The following extractors use this feature:
 #### nflplusreplay
 * `type`: Type(s) of game replays to extract. Valid types are: `full_game`, `full_game_spanish`, `condensed_game` and `all_22`. You can use `all` to extract all available replay types, which is the default
 
+#### ADN
+* `language`: Which language version to extract. Valid values are `fr` and `de`. If not provided, the language is extracted from the provided URL
```

Comment:
What was wrong with the first implementation? Why is this necessary when user can simply pass the appropriate URL?